kotlinpoet/src/main/java/com/squareup/kotlinpoet/Taggable.kt

```diff
@@ -145,13 +148,11 @@ public inline fun <reified T : Any> TypeAliasSpec.Builder.tag(tag: T?): TypeAlia
 public inline fun <reified T : Any> TypeSpec.Builder.tag(tag: T?): TypeSpec.Builder =
   tag(T::class, tag)
 
-internal fun Taggable.Builder<*>.buildTagMap(): TagMap = TagMap(LinkedHashMap(tags)) // Defensive copy
+internal fun Taggable.Builder<*>.buildTagMap(): TagMap = TagMap(tags.toImmutableMap())
```

Comment:
Not need since the factory method already does it?
```suggestion
internal fun Taggable.Builder<*>.buildTagMap(): TagMap = TagMap(tags)
```